assets/src/infrastructure/ui/layout/modal/Modal.tsx

```diff
@@ -36,11 +37,18 @@ const Modal: React.FC<ModalProps> = ({
 		);
 	}
 	return (
-		<ChakraModal isCentered scrollBehavior='inside' size='xl' {...props}>
+		<ChakraModal
+			// closeOnEsc={isClosable}
```

Comment:
it's a todo or just a comment?

Reply:
That's temporary during development, otherwise you will be locked down to fix the TAM errors 😄